anime-list-master.xml

```diff
@@ -1442,7 +1442,7 @@
   <anime anidbid="339" tvdbid="106641" defaulttvdbseason="2" episodeoffset="" tmdbid="" imdbid="">
     <name>Sore Yuke! Uchuu Senkan Yamamoto Youko II</name>
   </anime>
-  <anime anidbid="340" tvdbid="unknown" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
+  <anime anidbid="340" tvdbid="106641" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
```

Comment:
Second entry mapped to Season 1 without an offset.
This one is S01, IDs 338 and 339 are S00 covering eps. 1-3 and 4-6 respectvely

Reply:
Corrected

Sore Yuke! Uchuu Senkan Yamamoto Youko - Specials - Ep 1-3
Sore Yuke! Uchuu Senkan Yamamoto Youko II - Specials - Ep 3-6
Sore Yuke! Uchuu Senkan Yamamoto Youko (1999) - Season 1